CppExport/src/dependency_analysis/DependencyUnit.cpp

```diff
@@ -33,6 +33,26 @@ namespace CppExport {
 DependencyUnit::DependencyUnit(QString name, Model::Node* node)
 	: name_(name), node_(node), targets_(calculateTargets(node)) {}
 
+QSet<const DependencyUnit*> DependencyUnit::dependencies(const QList<const DependencyUnit*>& allUnits) const
+{
+	QSet<const DependencyUnit*> result;
+
+	for (auto dependencyTarget : targets_)
```

Comment:
This nested loop will get expensive, I can smell it :)
We should consider at some point caching which unit a target belongs to. I guess we don't need it for now, but definitely keep it in mind.